.github/workflows/dev-ecs.yml

```diff
@@ -24,9 +24,6 @@ jobs:
         uses: mbta/actions/commit-metadata@v1
       - uses: mbta/actions/build-push-ecr@v1
         id: build-push
-        env:
-          ERL_COOKIE: ${{ secrets.ERL_COOKIE }}
```

Comment:
Is this simply not needed anymore? Looks like the code that sets it here, as well as the code that uses it in the Dockerfile, was removed.

Reply:
Yep, apparently using Elixir releases eliminates the need to set this explicitly and it's generated randomly with `mix release`. It's used by Distillery at build time and I don't have too many details, but [this PR comment](https://github.com/mbta/signs_ui/pull/750#issuecomment-1096923145) as well as [this one](https://github.com/mbta/devops/pull/329#issuecomment-1100128716) mentions that its not needed with native releases.

I _think_ it's the value generated at `_build/prod/rel/signs_ui/releases/COOKIE` which is used in the shell script that's called to start the app